api/hex-bot.ts

```diff
@@ -1,12 +1,109 @@
 import { NowRequest, NowResponse } from '@now/node';
 import { Octokit } from '@octokit/rest';
-import Telegraf, { ContextMessageUpdate, Extra } from 'telegraf';
+import Telegraf, { ContextMessageUpdate, Extra} from 'telegraf';
 import { ExtraEditMessage } from 'telegraf/typings/telegram-types';
-import axios from 'axios';
+// import axios from 'axios';
 const fs = require('fs');
+const WizardScene = require('telegraf/scenes/wizard');
+const Stage = require('telegraf/stage');
+const session = require('telegraf/session')
+var talk = {};
 
 const PROD_ENV = process.env.NODE_ENV === 'production';
 const bot = new Telegraf(process.env.HEX_BOT_TOKEN || '');
+bot.use(session())
+
+const talkInfoWizard = new WizardScene('talk_info', 
+  (ctx) => {
+    talk = {};
+    if(ctx.from!.first_name){
+      var rawData = fs.readFileSync('members.json');
+      var memberData = JSON.parse(rawData);
+      const git_id = memberData[(ctx.from!.first_name)];
+      if(git_id){
+        ctx.reply('Awesome. Awesome. What would your DevTalk\'s name be? This field in compulsory.')
```

Comment:
one question you know right ctx.reply closes a function ... like in local host the commands running after ctx.reply is run but in production async ctx gets a value 
hences exits the function do test it out like in the DevTalk command have a dummy ctx.reply('hello') and check it will exit the function and wont reply for the remaining and will give 
```bash 
[error] reason: Client network socket disconnected before secure TLS connection was established
```

Reply:
I would stress on this try things out on a deployed server once I will merge this but have in mind this may not work as expected as I previously got a similar error.

Reply:
As far as I know, the `WizardScene` is meant to work like this. I will check, though.